### PR TITLE
Creating check out class

### PR DESCRIPTION
**Pull request by nadinefrawley**

### CODE REVIEW

**Comment by dylancoffey99** [comment]
We'll put the creation of checkout on hold until we know if it's really needed.

**Review by dylancoffey99 — COMMENTED** [review]
The creation of checkout is on hold until we know if it's really needed.

**Review by wasimghazal — APPROVED** [review]
I will approve the the review now and later if we need to do some change we can improve the file

**Pull request closed without merging**
